### PR TITLE
 Add Makefile, configuration files and scripts 

### PR DESCRIPTION
**Pull request by StefanJum**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@
 !.gitignore
 .*
 /a.out
```
Comment (razvand) [inline .gitignore]: Why are these here?

Ignore `build/`.

File: README.md [hunk 2]
```diff
@@ -1,84 +1,318 @@
...
 
-Before running the application, we need a virtual bridge to create a connection between the virtual machine and the host system.
-The HTTP client will run on the host and the HTTP server will run on the virtual machine.
-We assign address `172.44.0.1/24` to the bridge interface (pointing to the host) and we assign address `172.44.0.2/24` to the virtual machine, by passing boot arguments.
-The IP addresses are of our choosing, they can be changed to other values.
+On Ubuntu/Debian or other `apt`-based distributions, run the following command to install the requirements:
 
-We run the commands below to create and assign the IP address to the bridge `virbr0` (once again, our choice of name; any name works):
+```console
+$ apt-get install -y --no-install-recommends \
```
Comment (razvand) [inline README.md]:
```suggestion
$ sudo apt install -y --no-install-recommends \
```

File: README.md [hunk 3]
```diff
@@ -1,84 +1,318 @@
...
+     ```
+
+     Use the `tree` command to inspect the contents of the `.unikraft/` directory.
+     It should print something like this:
+
+     ```console
+     $ tree -F -L 2 .unikraft/
+     .unikraft/
+     |-- libs/
+     |   |-- lwip/
```
Comment (razvand) [inline README.md]:
```suggestion
     |   `-- lwip/
```

File: README.md [hunk 4]
```diff
@@ -1,84 +1,318 @@
...
+A web client (such as `wget`) is required to query the server.
+
+Open another console and use the `wget` command below to query the server:
+
+```console
 $ wget 172.44.0.2:8123
---2021-08-18 16:47:38--  http://172.44.0.2:8123/
-Connecting to 172.44.0.2:8123... connected.
+--2023-07-01 13:53:24--  http://172.44.0.2/
+Connecting to 172.44.0.2:80... connected.
```
Comment (razvand) [inline README.md]: It's this port `8123`? Why port `80`?

**Review by razvand — CHANGES_REQUESTED** [review]
See my comments?

**Comment by StefanJum** [comment]
Done @razvand

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request closed without merging**